NEWS

```diff
@@ -20,6 +20,10 @@ CHANGES WITH 248:
           The SYSEXT_LEVEL= field in os-release(5) may be used to specify the
           supported system extension level.
 
+        * A new ExtensionImages= unit setting can be used to apply the same
+          system extension image concept from systemd-sysext to services,
```

Comment:
to services → to the namespaced file hierarchy of specific services?

Reply:
changed